.deployment/publish-next-packages/Jenkinsfile

```diff
@@ -12,25 +12,6 @@ pipeline {
 
   stages {
 
-    stage('Checkout specified branch') {
```

Comment:
Is it now declared in the jenkins task? To be honest I like the declarative approach in the Jenkins file.

Reply:
yeah, it was defined in the jenkins task already. we made a second checkout for no reason.